### DISCUSSION TITLE
support of custom recoveries

### DISCUSSION THREAD

**Discussion by Klut** [mvt-project/mvt#215] [Ideas]
Hi there.
Thanks a lot for your hard work and this great piece of software!
I don't know if my idea is that "new" to you, but I'd like to see mvt-android checking all the data on the built-in flash memory of my devices through a USB cable. In the moment it doesn't support busybox, at least it's telling me exactly that when I start my devices in recovery mode with TWRP and let mvt run together with the check-adb option.
I see a big advantage by using the custom recovery, as all partitions can be mounted as root. The only other option to get a full scan of everything would be to install Magisk on the OS and I don't like that.

cheers,
Klut

**Comment by botherder** [comment]
Interesting idea. I would be happy to explore the option, but I don't have a twrp-enabled device at hand. I'll see if I manage to obtain one. What would it take to get busybox support properly?

**Reply by Klut** [reply]
I'm not a developer. I'm a systems integrator with some fantasy :)
I could imagine that the developers of TWRP could also be interested in this new feature. The best would be if you approach them to check all technical possibilities.

https://github.com/TeamWin/Team-Win-Recovery-Project

**Reply by anishsha1249-byte** [reply]
🧑‍🚀
